templates/support/system_tests/lewis_emulators/DEVICENAME/interfaces/stream_interface.py

```diff
@@ -6,11 +6,7 @@
 
 @has_log
 class DEVICENAMEStreamInterface(StreamInterface):
-
-    def __init__(self):
-        super(DEVICENAMEStreamInterface, self).__init__()
-        # Commands that we expect via serial during normal operation
-        self.commands = {
+    commands = {
             CmdBuilder(self.catch_all).arg("^#9.*$").build()  # Catch-all command for debugging
```

Comment:
This is now completely broken, is it not? If we're not in `__init__` anymore we no longer have `self` so can't reference `self.catch_all` as a function?

If we want to use this pattern we have to make it a string not function reference. But I'm confused why this is needed at all, both methods should work fine?